apps/condo/domains/ticket/access/TicketComment.js

```diff
@@ -69,7 +69,7 @@ async function canManageTicketComments ({ authentication: { item: user }, origin
             commentType = get(originalInput, 'type')
         } else if (operation === 'update' && itemId) {
             const comment = await getByCondition('TicketComment', { id: itemId, deletedAt: null })
-            if (!comment || comment.user !== user.id) return false
+            if (!comment || comment.createdBy !== user.id) return false
```

Comment:
it's better to use `user` rather than `createdBy`. Are you sure? Can we use logic like Ticket.client ?

Reply:
we can use logic with Ticket.client after https://github.com/open-condo-software/condo/pull/1381 will be in the master